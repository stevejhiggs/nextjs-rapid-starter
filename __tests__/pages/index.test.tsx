import React from 'react';
import { cleanup, render, screen } from '@testing-library/react';
import Index from '@/pages/index';

afterEach(cleanup);

describe('Pages -> /', () => {
  it('should match the snapshot', () => {
    const { container } = render(<Index />);
    expect(container).toMatchSnapshot();
  });

  it('Should have the right content', () => {
    render(<Index />);
    const titles = screen.getAllByRole('heading');
    expect(titles[0]).toHaveTextContent('Homepage');
  });
});
