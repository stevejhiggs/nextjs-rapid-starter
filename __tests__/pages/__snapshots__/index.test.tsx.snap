// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages -> / should match the snapshot 1`] = `
<div>
  <h2
    class="chakra-heading css-0"
  >
    Homepage
  </h2>
  <p>
    I am some nice paragraph content
  </p>
  <h2
    class="chakra-heading css-0"
  >
    Assets
  </h2>
  <p>
    Assets such as images can just be added to the /public directory
  </p>
  <img
    alt="spider"
    src="fileMock"
  />
</div>
`;
